core/src/main/java/hudson/slaves/WorkspaceList.java

```diff
@@ -312,6 +312,7 @@ public static FilePath tempDir(FilePath ws) {
 
     /**
      * The token that combines the project name and unique number to create unique workspace directory.
+     * @since TODO
      */
-    private static final String COMBINATOR = SystemProperties.getString(WorkspaceList.class.getName(),"@");
+    public static final String COMBINATOR = SystemProperties.getString(WorkspaceList.class.getName(),"@");
```

Comment:
used here: https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/3e5a08f079fc371119fcf1b76a60369724c2eb70/src/main/java/org/jenkinsci/plugins/workflow/libs/SCMSourceRetriever.java#L173, would be nice if it's an API so that everything uses the same separator.

Reply:
Also used [here](https://github.com/jenkinsci/durable-task-plugin/blob/b333ba1c708de63063321b8b029d1571a9de1a10/src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java#L390) in durable-task and [here](https://github.com/jenkinsci/workflow-cps-plugin/blob/455249a3b9b65450e4de0739a46668e2f5c1202e/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java#L194-L200) in workflow-cps (this is where the `@script` directories come from), and probably a few other places as well.